core/platform/search/elastic_search_services.py

```diff
@@ -107,47 +106,54 @@ def clear_index(index_name):
 
 
 def search(
-        query_string, index_name, offset=None,
-        limit=feconf.SEARCH_RESULTS_PAGE_SIZE, ids_only=False):
+        query_string, index_name, cursor=0,
+        size=feconf.SEARCH_RESULTS_PAGE_SIZE, ids_only=False):
     """Searches for documents matching the given query in the given index.
     NOTE: We cannot search through more than 10,000 pages from a search by
-    paginating using limit and offset. If the number of items to search through
+    paginating using size and offset. If the number of items to search through
     is greater that 10,000, use the elasticsearch scroll API instead.
 
     Args:
-        query_string: str. The search definition using Query DSL.
+        query_string: str. A JSON-encoded string representation of the
+            dictionary search definition that uses Query DSL.
```

Comment:
Can you include a link that explains "Query DSL"?

Reply:
Done.